services/site/src/components/homepage/CTASection.tsx

```diff
@@ -0,0 +1,46 @@
+import clsx from "clsx";
+import Link from "next/link";
+import React from "react";
+
+const CTASection: React.FunctionComponent<React.PropsWithChildren> = ({ children }) => {
+  return (
+    <section
+      className={clsx(
+        "mb-4 flex flex-col mt-2",
+        "xs:mt-6",
+        "sm:mb-12 sm:mt-8",
+        "md:mt-8 md:flex-row xl:mx-20",
+        "xl:mb-24 md:flex-row xl:items-center 2xl:mx-52",
+      )}
+    >
+      <div className={clsx("flex flex-col justify-center", "md:mb-4", "xl:shrink xl:grow xl:mb-0")}>
+        <h2 className={clsx("h4 leading-tight", "sm:h3 sm:leading-tight", "md:max-w-[20ch]", "xl:h2 xl:leading-tight xl:max-w-[20ch]")}>
```

Comment:
```suggestion
        <h2 className={clsx("h4 leading-tight", "sm:h3", "md:max-w-[20ch]", "xl:h2")}>
```

If you assign a class to md, it affects all sizes larger than md. You don't have to respecify it. You can save a lot of code by not setting these classes again unnecessarily.

Reply:
Sadly when I tried it, it often didn't work. The specific heading classes (h3, h4, h5,...) override the style, in this case the leading- and max-w-. therefore I have to assign it again.